trimmer_decorate.rb

```diff
@@ -0,0 +1,7 @@
+require './decorate'
```

Comment:
- [ ] Please make sure you test your code. This import is incorrect. There is no file named `decorate`. What you have is `decorator`. Kindly fix that.